From 8d80e470944bf10fff8645d0dd4a5860124f282d Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Tue, 3 Oct 2017 12:13:31 -0700
Subject: [PATCH] Don't put DESTDIR in the isomd5sum.pc file

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c8ea9b6..6edda1b 100644
--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,7 @@ install-devel:
 	install -m 0644 libcheckisomd5.h $(DESTDIR)/usr/include/
 	install -m 0644 libimplantisomd5.a $(DESTDIR)/usr/$(LIBDIR)
 	install -m 0644 libcheckisomd5.a $(DESTDIR)/usr/$(LIBDIR)
-	sed "s#@VERSION@#${VERSION}#g; s#@includedir@#${DESTDIR}/usr/include#g; s#@libdir@#${DESTDIR}/usr/${LIBDIR}#g" isomd5sum.pc.in > ${DESTDIR}/usr/share/pkgconfig/isomd5sum.pc
+	sed "s#@VERSION@#${VERSION}#g; s#@includedir@#/usr/include#g; s#@libdir@#/usr/${LIBDIR}#g" isomd5sum.pc.in > ${DESTDIR}/usr/share/pkgconfig/isomd5sum.pc
 
 clean:
 	rm -f *.o *.so *.pyc *.a .depend *~
-- 
2.13.5

