diff -up isomd5sum-1.0.4/Makefile.orig isomd5sum-1.0.4/Makefile
--- isomd5sum-1.0.4/Makefile.orig	2008-11-05 15:12:05.000000000 +0100
+++ isomd5sum-1.0.4/Makefile	2008-11-05 15:16:25.000000000 +0100
@@ -41,8 +41,8 @@ install: all
 	mkdir -p $(DESTDIR)/usr/share/man/man1
 	install -m 755 implantisomd5 $(DESTDIR)/usr/bin
 	install -m 755 checkisomd5 $(DESTDIR)/usr/bin
-	install -m 755 implantisomd5.1 $(DESTDIR)/usr/share/man/man1
-	install -m 755 checkisomd5.1 $(DESTDIR)/usr/share/man/man1
+	install -m 644 implantisomd5.1 $(DESTDIR)/usr/share/man/man1
+	install -m 644 checkisomd5.1 $(DESTDIR)/usr/share/man/man1
 	install -m 755 pyisomd5sum.so $(DESTDIR)/usr/$(LIBDIR)/$(PYTHON)/site-packages
 #	ln -s ../../bin/implantisomd5 $(DESTDIR)/usr/lib/anaconda-runtime/implantisomd5
 #	ln -s ../../bin/checkisomd5 $(DESTDIR)/usr/lib/anaconda-runtime/checkisomd5
